src/execution/index.ts

```
@@ -0,0 +1,79 @@
+import { ExecutionConfig, ExecutionResult } from './execution.types';
+import { IExecutionService } from './execution.interface';
+import fs from "fs";
```

ESLint

[error] Replace `"fs"` with `'fs'`

(prettier/prettier)